packages/pegasus/src/pegasus.js

```diff
@@ -21,11 +19,11 @@ import { makeCourierMaker, getCourierPK } from './courier.js';
 const DEFAULT_DENOM_TRANSFORMER = IBCSourceTraceDenomTransformer;
 const DEFAULT_TRANSFER_PROTOCOL = ICS20TransferProtocol;
 
-const TRANSFER_PROPOSAL_SHAPE = {
+const TRANSFER_PROPOSAL_SHAPE = M.splitRecord({
```

Comment:
consider normalizing casing,
```suggestion
const TransferProposalShape = M.splitRecord({
```